external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchSourceSuite.scala

```diff
@@ -1511,6 +1511,60 @@ abstract class KafkaSourceSuiteBase extends KafkaSourceTest {
       CheckAnswer(2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17)
     )
   }
+
+  for (mode <- Seq("continuous", "micro-batch")) {
+    val trigger = mode match {
```

Comment:
ah sorry I misunderstand our current test framework. I think we should do:
1. create a `protected def testSpark27494(trigger)...` in `KafkaSourceSuiteBase`
2. in `KafkaContinuousSourceSuite`, add `test("SPARK-27494 ...") { testSpark27494(ContinuousTrigger(100)) }`
2. in `KafkaMicroBatchSourceSuiteBase`, add `test("SPARK-27494 ...") { testSpark27494(ProcessingTime(100)) }`

Reply:
ok